dialog/input.go

```diff
@@ -0,0 +1,87 @@
+package dialog
+
+import (
+	"fyne.io/fyne"
+	"fyne.io/fyne/theme"
+	"fyne.io/fyne/widget"
+)
+
+// InputDialog is a variation of a dialog which prompts the user to enter some
+// text.
+type InputDialog struct {
+	*dialog
+
+	onConfirm func(string)
+
+	entry *widget.Entry
+
+	confirmButton *widget.Button
+}
+
+// SetText changes the current text value of the input dialog, this can
+// be useful for setting a default value.
+func (i *InputDialog) SetText(s string) {
+	i.entry.SetText(s)
+}
+
+// GetText returns the current text value entered in the input dialog.
+func (i *InputDialog) GetText() string {
+	return i.entry.Text
+}
+
+// SetPlaceholder defines the placeholder text for the entry
```

Comment:
This mismatch on capitalisation here is causing the Travis failures

Reply:
Fixed in e20f0f7f